dist/install-otp1

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
Same as `.github1` is this file an artifact or moving things around?

Reply:
Same reply as before 😄